lib/generators/draft/account/account_generator.rb

```diff
@@ -91,7 +91,8 @@ def authentication_helpers
 
         before_action(:load_current_#{singular_table_name.underscore})
         
-        # Uncomment this if you want to force #{plural_table_name} to sign in before any other actions
+        # Uncomment the "before_action" if you want to force #{plural_table_name} to sign in before any other actions
+        # Remember to also uncomment the "skip_before_action" in the corresponding authentication controller
```

Comment:
Perhaps the comment could be made even more direct with something like `in #{plural_table_name}_authentication_controller.rb`" rather than `in the corresponding authentication controller`?